src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverFactoryActivator.java

```diff
@@ -107,6 +109,19 @@
     @Reference
     EventAdmin eventAdmin;
 
+    /** Event admin. */
```

Comment:
Comment above seems to be wrong :) Please make the reference greedy and remove the commented out lines below

Reply:
Done